### DISCUSSION TITLE
Drop backpack blocks to mouse pointer position

### DISCUSSION THREAD

**Discussion by slyi** [ScratchAddons/ScratchAddons#5632] [Ideas] [answered]
When copying blocks from backpack into the editor, it drops the block to the top of the editor. Can the block be dropped at mouse pointer and the current scroll location in the editor. It take me ages to move the block all the way down to the desired drop location especially when doing for multiple block in back pack. When i wish to copy a block in as i have to drag from top of current visible editor, drop block, scroll down a page length, drag and drop again and again until re positioned to the desired location.

**Comment by CST1229** [comment, marked as answer]
By the way, you can use the Developer tools addon's copy-paste feature to make the dragging process a bit quicker.

**Reply by slyi** [reply]
Thanks that workaround is perfect, i just wish the backpack drop location would paste to the current editor scoll position by default
